source/includes/windows/quickstart.rst

```diff
@@ -10,7 +10,17 @@ Quickstart for Windows
 
 This procedure deploys a :ref:`Single-Node Single-Drive <minio-installation-comparison>` MinIO server onto |OS| for early development and evaluation of MinIO Object Storage and its S3-compatible API layer. 
 
-For instructions on deploying to production environments, see :ref:`deploy-minio-distributed`.
+.. note::
+
+   MinIO has limited support for multi-drive Windows deployments due to behaviors and limitations of the NTFS filesystem.
+
+   This documentation only covers Single-Node Single-Drive deployment topologies on Windows.
+   Use Linux hosts for deploying Single-Node Multi-Drive or Multi-Node Multi-Drive MinIO topologies.
+
+Use Windows-based MinIO deployments for early development and evaluation.
+MinIO strongly recommends Linux (RHEL, Ubuntu) systems for long-term development and production environments.
+
+MinIO supports non-EOL Windows versions (Windows 10, Windows Server 2016+). 
 
 Prerequisites
 -------------
```

Comment:
- Line 83, should update the port to the Console port (`9090`).
- Line 84 is confusing when rendered. I suggest making the :guilabel: section a code block instead and splitting up the paragraph.
   Or, just making it monotype instead of guilabel might be sufficient.